platform/vpp/vppbld/Makefile

```diff
@@ -93,8 +93,13 @@ docker_build:  vppbld-docker/Dockerfile-$(BLDENV)
 	rm -rf debs
 	popd
 
-$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% : docker_build repo_clone
-	$(VPP_DOCKER_RUN) /run_build.sh
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :  repo_clone
+
+	pushd $(VPP_REPO_DIR)
+	make UNATTENDED=y PLATFORM=vpp install-dep 
```

Comment:
With this change, install-dep will install some packages to slave docker. It may cause build and runtime issues to other components. So I chose to use its own container to build vpp

Reply:
I didn't think it through.